test/retrieve.jl

```diff
@@ -0,0 +1,32 @@
+using CDSAPI
+using Test
+
+@testset "ERA5 data" begin
+    name = "reanalysis-era5-pressure-levels-monthly-means"
+    params = """{
+            'format': 'grib',
+            'product_type': 'monthly_averaged_ensemble_members_by_hour_of_day',
+            'variable': 'divergence',
+            'pressure_level': '775',
+            'year': '2020',
+            'month': '03',
+            'time': '12:00',
+        }"""
+
+    if !isdir("data")
+        mkdir("data")
```

Comment:
I wouldn't create a directory in the user's call site. Please use the following strategy with a `data` directory inside the test folder:

```julia
datadir = joinpath(@__DIR__,"data")
```

You can create the directory and commit a file like a "README.md" inside it if Git doesn't allow to commit empty directories. Then at test time you just need to download data in the folder assuming it already exists.

Reply:
done and tested.